filament/backend/CMakeLists.txt

```diff
@@ -242,11 +242,29 @@ endif()
 
 if (FILAMENT_SUPPORTS_WEBGPU)
     list(APPEND SRCS
+            include/backend/platforms/WebGPUPlatform.h
+            src/webgpu/platform/WebGPUPlatform.cpp
+            src/webgpu/WebGPUConstants.h
             src/webgpu/WebGPUDriver.cpp
             src/webgpu/WebGPUDriver.h
-            src/webgpu/WebGPUPlatform.cpp
-            src/webgpu/WebGPUPlatform.h
     )
+    if (LINUX OR WIN32)
+        list(APPEND SRCS src/webgpu/platform/WebGPUPlatformLinuxWindows.cpp)
```

Comment:
probably better to split linux and windows into two.

Sorry for the counter-example in the vulkan backend.  I kind of regret combining the two.